lib/proofer/applicant.rb

```diff
@@ -1,5 +1,6 @@
 module Proofer
   class Applicant
+    attr_accessor :uuid
```

Comment:
I had originally used separate lines because I was grouping the attributes together by category, and they were very much in flux as I evaluated multiple different vendors to define the set of needed attributes.